lib/chef/knife/core/windows_bootstrap_context.rb

```diff
@@ -158,14 +158,35 @@ def start_chef
           start_chef << "chef-client -c c:/chef/client.rb -j c:/chef/first-boot.json#{bootstrap_environment_option}\n"
         end
 
+        #
+        # Provide the query arguments to a URL that will be appeded to the URL in #msi_url().  This is used
+        # to get the latest version of Chef Infra Client via omnitruck
+        # By default, bootstrap will look for the latest version of the currently-running major release of on stable.
+        #
+        # @return [String] query arguments to append to the download request.
         def latest_current_windows_chef_version_query
+          # NOTE: Changes here should also be reflected in Knife::Core::BootstrapContext#latest_current_chef_version_string
+          use_current_channel = (@config[:channel] == "current")
+          installer_version_string = use_current_channel ? "&prerelease=true" : ""
```

Comment:
fixing this to pass a channel instead of just prerelease means changing the URL down in `msi_url` to use omnitruck instead of whatever that chef.io redirect is.

https://docs.chef.io/api_omnitruck.html

I don't know how much time we want to spend on that though, because what we really need to do is replace all this with https://github.com/chef/mixlib-install